crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/validation/validator/MeansAssessmentValidationProcessor.java

```diff
@@ -59,6 +61,12 @@ public Optional<Void> validate(MeansAssessmentRequestDTO requestDTO, AssessmentR
                 throw new ValidationException(MSG_FULL_ASSESSMENT_DATE_REQUIRED);
             }
         }
+
+        if (AssessmentRequestType.UPDATE.equals(requestType) &&
```

Comment:
There is already a check for the CREATE request type. This should form the else block of that conditional rather than being separate.